### PR TITLE
[RFR] Integrate ra-realtime

### PR DESCRIPTION
**Pull request by djhi**

### CODE REVIEW

File: packages/ra-realtime/src/index.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                yield [
+                    put({
+                        type: `${type}_LOADING`,
+                        payload,
+                        meta: omit(meta, 'fetch'),
+                    }),
+                    put({ type: FETCH_START }),
+                ];
+
+                const aorAction = yield call(
```
Comment (fzaninotto) [inline packages/ra-realtime/src/index.js]: ra-

File: packages/ra-realtime/src/buildAorAction.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { FETCH_END } from 'react-admin';
```
Comment (fzaninotto) [inline packages/ra-realtime/src/buildAorAction.js]: rename file to remove `aor`

File: packages/ra-realtime/src/index.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        }
+    };
+
+export const watchLocationChangeFactory = watchCrudActions =>
+    function* watchLocationChange() {
+        yield takeLatest([CRUD_GET_LIST, CRUD_GET_ONE], watchCrudActions);
+    };
+
+export default observeQuery =>
+    function* aorGraphQlSaga() {
```
Comment (fzaninotto) [inline packages/ra-realtime/src/index.js]: ra-

File: packages/ra-realtime/src/buildAorAction.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+export default (
+    { type, payload, meta: { fetch: restType, ...meta } },
+    parsedApolloQueryResult
```
Comment (fzaninotto) [inline packages/ra-realtime/src/buildAorAction.js]: why mention Apollo` here? Isn't this packagfe supposed to be independent from Apollo?

File: packages/ra-realtime/src/index.js (hunk 3 above)
Comment (fzaninotto) [reply packages/ra-realtime/src/index.js]: also, why mention GraphQL here?

File: packages/ra-realtime/src/queryObserver.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import { END } from 'redux-saga';
+
+export default emitter => ({
+    complete() {
+        emitter(END);
+    },
+    error() {
+        emitter(END);
+    },
+    next(apolloQueryResult) {
```
Comment (fzaninotto) [inline packages/ra-realtime/src/queryObserver.js]: please remove mention of apollo

File: packages/ra-realtime/src/buildAorAction.js (hunk 4 above)
Comment (djhi) [reply packages/ra-realtime/src/buildAorAction.js]: Probably a left-over from when it was integrated in the simple graphql client...

File: packages/ra-realtime/src/index.js (hunk 1 above)
Comment (djhi) [reply packages/ra-realtime/src/index.js]: raaaaah !

**Pull request merged**